components/engine_rocks/src/table_properties.rs

```diff
@@ -33,6 +23,16 @@ impl TablePropertiesExt for RocksEngine {
         let raw = raw.map_err(Error::Engine)?;
         Ok(RocksTablePropertiesCollection::from_raw(raw))
     }
+
+    pub fn get_range_properties_cf(
+        &self,
+        cfname: &str,
+        start_key: &[u8],
+        end_key: &[u8],
+    ) -> Result<RocksTablePropertiesCollection> {
+        let range = Range::new(start_key, end_key);
+        self.get_properties_of_tables_in_range(cfname, &[range])
+    }
 }
 
 pub struct RocksTablePropertiesCollection(rc::TablePropertiesCollection);
```

Comment:
Do we need these wrappers anymore?

Reply:
Not strictly, but there was some reason that I have forgotten now for leaving them there. I'll try to remove them completely. Thanks for the review.

Reply:
OK, I've removed all these wrappers.